spec/pg_eventstore/queries/subscription_queries_spec.rb

```diff
@@ -0,0 +1,276 @@
+# frozen_string_literal: true
+
+RSpec.describe PgEventstore::SubscriptionQueries do
+  let(:instance) { described_class.new(PgEventstore.connection) }
+
+  describe '#find_by' do
+    subject { instance.find_by(attrs) }
+
+    let(:attrs) { { set: 'Foo', name: 'Bar' } }
+
+    describe 'when subscription exists' do
+      let!(:subscription) { SubscriptionsHelper.create(**attrs) }
+
+      it 'returns its attributes' do
+        is_expected.to eq(subscription.options_hash)
+      end
+    end
+
+    describe 'when subscription does not exist' do
+      it { is_expected.to be_nil }
+    end
+  end
+
+  describe '#find!' do
+    subject { instance.find!(id) }
+
+    let(:id) { 123 }
```

Comment:
I would move it to the proper context

Reply:
Same as above. `id` is used in subject, thus it should be on the `subject`'s level